Auth0/WebAuthError.swift

```diff
@@ -8,18 +8,14 @@ import Foundation
  - CannotDismissWebAuthController: When trying to dismiss WebAuth controller, no presenter controller could be found.
  - UserCancelled:                  User cancelled the web-based authentication, e.g. tapped the "Done" button in SFSafariViewController
  - PKCENotAllowed:                 PKCE for the supplied Auth0 ClientId was not allowed. You need to set the `Token Endpoint Authentication Method` to `None` in your Auth0 Dashboard
- - noNonceProvided:                A nonce value must be provided to use the response option of id_token
- - invalidIdTokenNonce:            Failed to match token nonce with request nonce
  - missingAccessToken:             access_token missing in response
  */
 public enum WebAuthError: CustomNSError {
     case noBundleIdentifierFound
     case cannotDismissWebAuthController
     case userCancelled
     case pkceNotAllowed(String)
-    case noNonceProvided
     case missingResponseParam(String)
-    case invalidIdTokenNonce // TODO: Remove on the next major
```

Comment:
This one stopped being used when the ID Token validation was implemented, adding errors of its own.